openquake/mbt/tools/fault_modeler/fault_modeling_utils.py

```diff
@@ -2871,10 +2956,12 @@ def calc_youngs_coppersmith_mfd_from_fault_params(
     :type M_char:
         float
 
-    :param M_min:
-        Minimum magnitude in the fault's magnitude-frequency distribution.
+    :param M_cli:
+        In a 'Youngs-Coppersmith' we calculate the MFD using two magnitude
+        extremes: M_cli and M_char. M_cli is the value of magnitude above
+        which we effectively compute the rates.
 
-    :type M_min:
+    :type M_cli:
```

Comment:
Note that the name of the variables should be just lower case.

Reply:
lgtm